packages/types/src/electra/types.ts

```diff
@@ -34,3 +34,10 @@ export type LightClientUpdate = ValueOf<typeof ssz.LightClientUpdate>;
 export type LightClientFinalityUpdate = ValueOf<typeof ssz.LightClientFinalityUpdate>;
 export type LightClientOptimisticUpdate = ValueOf<typeof ssz.LightClientOptimisticUpdate>;
 export type LightClientStore = ValueOf<typeof ssz.LightClientStore>;
+
+export type Attestation = ValueOf<typeof ssz.Attestation>;
+export type IndexedAttestation = ValueOf<typeof ssz.IndexedAttestation>;
+export type AggregateAndProof = ValueOf<typeof ssz.AggregateAndProof>;
+export type SignedAggregateAndProof = ValueOf<typeof ssz.SignedAggregateAndProof>;
+export type AttesterSlashing = ValueOf<typeof ssz.AttesterSlashing>;
+export type IndexedAttestationBigint = ValueOf<typeof ssz.IndexedAttestationBigint>;
```

Comment:
Can you ensure the exported types here are in the same order as the exported sszTypes?